### PR TITLE
Switching from grid to list and list to grid

### PR DESCRIPTION
**Pull request by nikunjsdev**
Switching from grid to list and list to grid

### CODE REVIEW

File: src/components/Collection/GridCollection.tsx [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-export const GridCollection = () => {
-  return(
-    <div className="collection-wrapper__main collection-wrapper__grid">
-        <ReactSVG path={gridIcon} className="collection-wrapper__main-icon"/>
-        <span className="collection-wrapper__main-name">Grid</span>
+export const GridCollection = ({ handleViewChange }) => {
+  return (
+    <div
+      className="collection-wrapper__main collection-wrapper__grid"
+      onClick={() => handleViewChange("grid")}
```
Comment (zshanhui) [inline src/components/Collection/GridCollection.tsx]: Should be capitalized for constants: "GRID",

And put in a Typescript enum: `enum GridViewTypes`

File: src/views/BrowseCabinets/scss/index.scss [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
 .browse-cabinet {
   padding: 17px;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss]: Use `rem` units or `$spacer`

File: src/views/BrowseCabinets/scss/index.scss [hunk 3]
```diff
@@ -15,6 +14,7 @@
       .cart-main {
         position: relative;
         margin-right: 20px;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss]: Use `rem` units or `$spacer`

File: src/views/BrowseCabinets/scss/index.scss [hunk 4]
```diff
@@ -15,6 +14,7 @@
...
+
         &-item {
           position: absolute;
           top: -4px;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss]: Use `rem` units or `$spacer`

File: src/views/BrowseCabinets/scss/index.scss [hunk 5]
```diff
@@ -35,14 +35,14 @@
 
       .cart-icon {
         .a {
-          stroke: #4E4E4E;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss:38]: Use variables

File: src/views/BrowseCabinets/scss/index.scss [hunk 6]
```diff
@@ -35,14 +35,14 @@
...
+          stroke: #4e4e4e;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss]: Use variables

File: src/views/BrowseCabinets/scss/index.scss [hunk 7]
```diff
@@ -35,14 +35,14 @@
...
         }
       }
 
       .search-icon {
         svg {
           & path:first-child {
-            fill: #4E4E4E;
+            fill: #4e4e4e;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss]: Use variables

File: src/views/BrowseCabinets/scss/index.scss [hunk 8]
```diff
@@ -62,7 +62,7 @@
     margin-bottom: 37px;
 
     h3 {
-      color: #B09A51;
```
Comment (zshanhui) [inline src/views/BrowseCabinets/scss/index.scss:65]: Use variables

File: src/views/Samples/Page.tsx [hunk 9]
```diff
@@ -1,21 +1,24 @@
...
 
 import "./scss/index.scss";
 
 const Page = props => {
   const { data, history } = props;
+  const [view, changeView] = useState("grid");
   const handleBack = () => {
     history.push("/");
   };
   const regex = /\s|_|(?=[A-Z])/;
```
Comment (zshanhui) [inline src/views/Samples/Page.tsx]: Delete if not used.

**Review by zshanhui — CHANGES_REQUESTED** [review]
@nikunjsdev Just a few changes and then good to merge.

File: src/components/Collection/GridCollection.tsx (hunk 1 above)
Comment (nikunjsdev) [reply src/components/Collection/GridCollection.tsx]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 2 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 3 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 4 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 5 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss:38]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 6 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 7 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss]: Done

File: src/views/BrowseCabinets/scss/index.scss (hunk 8 above)
Comment (nikunjsdev) [reply src/views/BrowseCabinets/scss/index.scss:65]: Done

File: src/views/Samples/Page.tsx (hunk 9 above)
Comment (nikunjsdev) [reply src/views/Samples/Page.tsx]: It's getting used

Comment (zshanhui) [reply src/views/Samples/Page.tsx]: I think I meant this line: `const regex = /\s|_|(?=[A-Z])/; `

**Review by zshanhui — DISMISSED** [review]

**Comment by zshanhui** [comment]
@nikunjsdev 
lgtm. There's some conflicts, but you can merge when you're ready.

**Review by zshanhui — APPROVED** [review]
lgtm

**Pull request merged**